apps/examples/testcase/le_tc/filesystem/fs_main.c

```diff
@@ -5593,6 +5593,9 @@ int tc_filesystem_main(int argc, char *argv[])
 	tc_libc_stdio_ungetc_p();
 	tc_libc_stdio_ungetc_invalid_arg_n();
 	tc_libc_stdio_zeroinstream_p();
+#ifdef CONFIG_ITC_FS
+	itc_fs_main();
```

Comment:
I think you'd better call itc after finishing the utcs.

Reply:
Change to call itcs last.